### PR TITLE
Add javax.annotation.Generated annotation to generated classes

### PR DESCRIPTION
**Pull request by sav007**
Annotate generated classes with `javax.annotation.Generated` to indicated that source is been generated by apollo

@digitalbuddha

### CODE REVIEW

File: apollo-compiler/src/main/kotlin/com/apollostack/compiler/Annotations.kt [hunk 1]
```diff
@@ -1,11 +1,15 @@
...
+import javax.annotation.Generated
 import javax.annotation.Nonnull
 import javax.annotation.Nullable
 
 object Annotations {
   val NULLABLE: AnnotationSpec = AnnotationSpec.builder(Nullable::class.java).build()
   val NONNULL: AnnotationSpec = AnnotationSpec.builder(Nonnull::class.java).build()
   val OVERRIDE: AnnotationSpec = AnnotationSpec.builder(Override::class.java).build()
+  val GENERATED_BY_APOLLO: AnnotationSpec = AnnotationSpec.builder(Generated::class.java)
+      .addMember("value", CodeBlock.of("\$S", "Apollo GraphQl")).build()
```
Comment (digitalbuddha) [inline apollo-compiler/src/main/kotlin/com/apollostack/compiler/Annotations.kt]: @martijnwalraven should we do ApolloData as the string?

File: apollo-compiler/src/main/kotlin/com/apollostack/compiler/OperationTypeSpecBuilder.kt [hunk 2]
```diff
@@ -13,7 +13,9 @@ class OperationTypeSpecBuilder(
 
   override fun toTypeSpec(abstractClass: Boolean, reservedTypeNames: List<String>,
       typeDeclarations: List<TypeDeclaration>, fragmentsPackage: String, typesPackage: String): TypeSpec {
-    return TypeSpec.classBuilder(OPERATION_TYPE_NAME)
+    return TypeSpec
+        .classBuilder(OPERATION_TYPE_NAME)
```
Comment (digitalbuddha) [inline apollo-compiler/src/main/kotlin/com/apollostack/compiler/OperationTypeSpecBuilder.kt]: Spacing is inconsistent. In last file   .classBuilder was on previous line

File: apollo-compiler/src/main/kotlin/com/apollostack/compiler/Annotations.kt (hunk 1 above)
Comment (digitalbuddha) [inline apollo-compiler/src/main/kotlin/com/apollostack/compiler/Annotations.kt]: Of we keep GraphQL why is the Q capital  but not the L. Not sure if Nitpick

Comment (martijnwalraven) [inline apollo-compiler/src/main/kotlin/com/apollostack/compiler/Annotations.kt]: Yeah, I think Apollo GraphQL would be fine.

**Pull request merged**